### DISCUSSION TITLE
"sanitize" 和 "code preview, code row number" 好像有衝突？

### DISCUSSION THREAD

**Discussion by Canis-Infinity** [imzbf/md-editor-rt#110] [General]
```js
<MdEditor
    modelValue={text}
    onChange={setText}
    language="zh-TW"
    theme={theme}
    showCodeRowNumber={true}
    previewTheme="github"
    sanitize={sanitize}
    formatCopiedText={formatCopiedText}
/>
```
我是這樣寫的，加上 `sanitize` 之後，就會像這樣沒有高亮和行數
<img width="757" alt="image" src="https://github.com/imzbf/md-editor-rt/assets/16958052/d3b23d25-767c-4420-8a94-5a701b98c931">

這是沒有加 `sanitize` 的版本 ，是正常的
<img width="756" alt="image" src="https://github.com/imzbf/md-editor-rt/assets/16958052/ab68b842-fe2c-42e0-b370-36c9f8fbb2fc">

請問一下，是我的寫法有問題嗎？

**Comment by imzbf** [comment]
不太清楚你的`sanitize`做了什么，但是在3.0版本以后，处理危险的内容已经在编辑器内容所依赖的库中实现了，所以如果不是特别需要，建议这边不设置该属性
